Makefile

```
@@ -2,11 +2,11 @@ all: dev
 
 .PHONY: dev
 dev:
-	@docker compose up --build
+	@docker compose -f docker-compose.dev.yml up --build
 
```

Отдельно фронт есть в режиме дев и прод, а все контейнеры только в дев. Мб, стоит добавить прод для всех контейнеров:
```suggestion

.PHONY: prod
prod:
	@docker compose up --build
	
```

---

И, кстати, возможно стоит сделать `prod` режимом по умолчанию (при запуске с помощью `make`), т.к. сейчас `docker compose up` поднимает прод версию.